Lib/fontTools/ttLib/tables/grUtils.py

```diff
@@ -13,7 +13,7 @@ def decompress(data):
     if scheme == 0:
         pass
     elif scheme == 1 and lz4:
-        res = lz4.decompress(struct.pack("<L", size) + data[8:])
+        res = lz4.block.decompress(struct.pack("<L", size) + data[8:])
```

Comment:
I must confess, this is the first time I notice that this Graphite tables in fontTools require `lz4` compression library
Is this the one you are using here?
https://pypi.org/project/lz4/
https://python-lz4.readthedocs.io/en/stable/

If so we need to document it in the README, like we do with other optional dependencies, and also add a new `[graphite2]` (or just graphite) "extra" in setup.py, in the `extras_require` setup option.

Reply:
yes. Will do.